tests/src/test/scala/cats/tests/EitherTests.scala

```diff
@@ -17,6 +18,11 @@ class EitherTests extends CatsSuite {
   val monad = implicitly[Monad[Either[Int, ?]]]
   val show = implicitly[Show[Either[Int, String]]]
 
+  val orderLaws = OrderLaws[Either[Int, String]]
+  checkAll("Either[Int, String]", orderLaws.eqv)
+  checkAll("Either[Int, String]", orderLaws.partialOrder(partialOrder))
+  checkAll("Either[Int, String]", orderLaws.order(order))
```

Comment:
Ahh I see :-) I think I actually prefer your way over figuring our which types happen to have the instances I want